frontend/www/js/omegaup/problem/collection_list.ts

```diff
@@ -72,13 +72,15 @@ OmegaUp.on('ready', () => {
             columnName: string,
             sortOrder: omegaup.SortOrder,
             difficulty: string,
+            tag: string[],
```

Comment:
nit:

```suggestion
            tags: string[],
```

esto debería ir en plural

Reply:
Lo que pasa es que el parámetro es así :s por ejemplo al seleccionar dos etiquetas en los parámetros de la url aparece algo como `tag[]=problemTagArrays&tag[]=problemTagMatrices` tal como funciona en `/problem`. Creo que se requieren cambios en el controlador para que acepte `tags[]=problemTagArrays&tags[]=problemTagMatrices` ¿se debería hacer?

Reply:
> Lo que pasa es que el parámetro es así :s por ejemplo al seleccionar dos etiquetas en los parámetros de la url aparece algo como `tag[]=problemTagArrays&tag[]=problemTagMatrices` tal como funciona en `/problem`. Creo que se requieren cambios en el controlador para que acepte `tags[]=problemTagArrays&tags[]=problemTagMatrices` ¿se debería hacer?

nah, déjalo así.